numpy/core/arrayprint.py

```diff
@@ -51,18 +52,18 @@
     _MAXINT = sys.maxint
     _MININT = -sys.maxint - 1
 
+
 def product(x, y):
     return x*y
 
-_summaryEdgeItems = 3     # repr N leading and trailing items of each dimension
-_summaryThreshold = 1000  # total items > triggers array summarization
-
-_float_output_precision = 8
-_float_output_suppress_small = False
-_line_width = 75
-_nan_str = 'nan'
-_inf_str = 'inf'
-_formatter = None  # formatting function for array elements
+_options = dict(edgeitems=3,     # repr N leading and trailing items of each dimension
+                threshold=1000,  # total items > triggers array summarization
+                precision=8,
+                suppress=False,
```

Comment:
Can we add a comment here like 'for small floating values', since we lost that information in renaming this?